invenio_rdm_migrator/load/__init__.py

```diff
@@ -8,11 +8,10 @@
 """Invenio RDM migration load module."""
 
 from .base import Load
-from .postgresql import ExistingDataTableGenerator, PostgreSQLCopyLoad, TableGenerator
+from .postgresql import PostgreSQLCopyLoad, PostgreSQLExecute
 
 __all__ = (
-    "ExistingDataTableGenerator",
     "Load",
     "PostgreSQLCopyLoad",
-    "TableGenerator",
```

Comment:
The load module exposes:

- Base class.
- Implementations of `Base`. (hesitating on exposing this or forcing developers to import from `.postgresql`)

It does not make sense to expose table generators since now there are many types and the developer will need to subclass some of them to implement (at least) `_generate_rows`. They can work with the submodule.